pkg/controller/nodekeeper/nodekeeper_controller.go

```diff
@@ -142,7 +143,11 @@ func (r *ReconcileNodeKeeper) Reconcile(ctx context.Context, request reconcile.R
 	if err != nil {
 		return reconcile.Result{}, nil
 	}
-	cfm := r.configManagerBuilder.New(r.client, operatorNamespace)
+
+	target := config.CMTarget{Namespace: operatorNamespace}
```

Comment:
I see we have multiple places with `util.GetOperatorNamespace()`
And the put the value to the CMTarget.

Can we just move the `util.GetOperatorNamespace()` to `config/config.go`? so that we can call the `NewCMTarget()` directly.

Reply:
Move it so we call `config.GetOperatorNamespace()` ?

Reply:
move the `util.GetOperatorNamespace()` inside the `NewCMTarget()` so that we do not need to call it separately

Reply:
what if we want to pass an explicit namespace in future?

Reply:
I think it should still be able to read from args as what it current do? just set the default from the return of `util.GetOperatorNamespace()` instead of the hardcoded `OperatorNamespace`

Reply:
sounds good. Resolved, squashed and pushed. Please review.